packages/crypto/tests/rootberg_secp256k1.rs

```diff
@@ -0,0 +1,164 @@
+use cosmwasm_crypto::{secp256k1_recover_pubkey, secp256k1_verify};
+use serde::Deserialize;
+
+// See ./testdata/wycheproof/README.md for how to get/update those files
```

Comment:
This still references the old path

```suggestion
// See ./testdata/rootberg/README.md for how to get/update those files
```